### DISCUSSION TITLE
Subdomain is not indexing in hostname-rewrite example

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#263] [Help] [answered]
**Hi everyone,**
I'm using the hostname-rewrites middleware example, however only the subdomains are not showing up on Google even though my project has a server side sitemap and robots.

Could someone explain to me how to solve the indexing?

[https://github.com/vercel/examples/tree/main/edge-functions/hostname-rewrites](url)
### pages/robots.txt.js
![image](https://user-images.githubusercontent.com/67167087/151394360-066bddfa-a4f3-4f1c-a512-aee4c789c9f0.png)

### pages/sitemap.xml.js
![image](https://user-images.githubusercontent.com/67167087/151394546-5d7716ed-64e0-4836-964d-03da9833613b.png)

**Comment by Pieparker** [comment, marked as answer]
How Google crawls pages is a bit of a hidden art, to be honest! Your best bet would be to [ask Google to crawl your site](https://developers.google.com/search/docs/advanced/crawling/ask-google-to-recrawl).

**Reply by leerob** [reply]
We also have a course on SEO here: https://nextjs.org/learn/seo
